tox.ini

```diff
@@ -11,6 +11,8 @@ deps =
 
 [testenv:py3]
 commands = nosetests -v monocle/tests
+         python ./monocle/gerrit/review.py --base-url https://gerrit-review.googlesource.com --repository gerrit --id 246332 --output-dir /tmp
```

Comment:
Why is this needed ? Why not provide the raw.json and extracted.json in fixtures ?

Reply:
I remember we used to lose these commands. I think we should add a new tox command to add all the subcommand to fetch github and gerrit fixtures. This way we keep the commands, and it is on demand. Here I don't this that's ok to fetch the data when running the test suite.